packages/typespec-ts/test/unit/modelsGenerator.spec.ts

```diff
@@ -236,9 +239,12 @@ describe("Input/output model type", () => {
     it("should handle enum -> string_literals", async () => {
       const cadlTypeDefinition = `
       #suppress "@azure-tools/typespec-azure-core/use-extensible-enum" "for test"
+      #suppress "@azure-tools/typespec-azure-core/documentation-required" "for test"
       @fixed
       enum TranslationLanguageValues {
+        @doc("English")
```

Comment:
why does this has no impact on the generated code of this test ?

Reply:
I am we should generate the doc right ?

Reply:
We will generate the `"English" | "Chinese"` directly no descriptions would be used here.